### ISSUE TITLE
aurelia 2 support

### ISSUE THREAD

**Issue by nmocruz** [code-chris/aurelia-knockout#30]
any plans for support aurelia version 2?

**Comment by ckotzbauer** [comment]
No, I hadn't time yet to look what would be necessary to add support for Aurelia 2. And I don't know too, if the implementations for both versions can live alongside...
Maybe @fkleuver can have a look and provide some information?

**Comment by fkleuver** [comment]
Thanks for pinging me, I'll add this plugin to the backwards compat checklist to make sure there's a way to make it work. Will get back to you post-alpha :)

**Comment by ckotzbauer** [comment]
Great, thank you! @fkleuver

**Comment by nmocruz** [comment]
I notice that version 2 has a binding transformer. 
Seems very hard to parse ko bindings, but it may be possible to process ko templates using Aurelia. Of course bindings with js expression and ko context access can be complex
